new-readme.md

```diff
@@ -3,30 +3,31 @@
 
-reSolve is a framework for developing apps based on **CQRS**, **Domain-Driven Design** and **Event Sourcing** using **React + Redux** on the client. It negates differences between your domain and technical languages, and allows you to focus on application code.
+A JavaScript framework for **CQRS**, **Domain-Driven Design** and **Event Sourcing** with good support for **React** and **Redux**. **reSolve** provides a new way to describe app's core logic and fully separate it from an auxiliary code.
 
 <p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-technologies.png"></p>
 
-* **2 minute** easy [setup](#installation)
-* **Free and open source.** reSolve is free to use under the [MIT license](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
-* **Flexibility.** Great opportunities with [clear API](#documentation).
+* The [Installation](#installation) is fully automated and takes 2 minutes.
+* [Free and open source](https://github.com/reimagined/resolve/blob/master/LICENSE.md).
+* Flexible and clear [API](#documentation).
 
+<!-- TODO: text="Try it now!" -->
 [<p align="center"><img src="https://github.com/reimagined/resolve/blob/feature/new_readme/readme-start-button.png"></p>](#getting-started)
 
 ## **📑 Table of Contents**
-* #### ️️️[🏗️ Why reSolve](#why-resolve)
-* #### [🚀 Getting Started](#getting-started)
+* **[🏗️ Why reSolve](#why-resolve)**
+* **[🚀 Getting Started](#getting-started)**
     * [Installation](#installation)
     * [Tutorials](#tutorials)
     * [Examples](#examples)
-* #### [📚 Documentation](#documentation)
+* **[📚 Documentation](#documentation)**
     * [Compatibility](#compatibility)
     * [Bacic Concepts](#basic-concepts)
-* #### [📢 Get in Touch](#get-in-touch)
+* **[📢 Get in Touch](#get-in-touch)**
 
 ## <a name="why-resolve">**🏗️ Why reSolve**</a>
 
-With reSolve you can easy and fast create different level difficulty applications. It could be applications like simple [ToDo List](#todo-list), or some reactive applications like StackOverflow, with distributed, highload and serverless architecture. Thanks to stack of technologies (CQRS, Event Sourcing, Domain-Driven Design, React + Redux) your application would be easy scalable, maintainable, so you can focus on application essence.
+With **reSolve** you can create applications of any complexity level more efficiently. A single approach is used for simple [ToDo List](#todo-list) and for a complex reactive application like [Hacker News](https://github.com/reimagined/hacker-news-resolve) with distributed serverless architecture ready for high load. The technologies stack (CQRS, Event Sourcing, Domain-Driven Design, React + Redux) makes any **reSolve** application scalable and maintainable, so you can focus on business logic instead of an auxiliary code.
```

Comment:
HackerNews with distributed serverless architecture? No. It's stackoverflow, not hn :)

Reply:
OK, do we have an example for stackoverflow?

Reply:
If not, I think we should not mention any existing service. StackOverflow is not based on reSolve now.

Reply:
Because we want to define types of applications that can be made using reSovle. We can just write "similar stackoverflow", "like stackoverflow", "for example, such as stackoverflow", or something like this. We guess the SO application is our etalon of difficult application.

Reply:
We have one thing that we can't do now with reSolve - it's serverless. But it'a a common feature that we can plan to add, currently you can just delete 'serverless' word. You can make the SO now but without serverless.

Reply:
There's no proof that I can make SO. Unexpected difficulties are very possible with such a big service. For instance, the performance.

Reply:
No. HN is too simple application. We want to leave this text, we'll make the SO example little later.

Reply:
I'll change this.